src/gunicorn_prometheus_exporter/backend/core/values.py

```
@@ -0,0 +1,137 @@
+    
+    Args:
+        redis_client: Redis client instance
+        redis_key_prefix: Prefix for Redis keys
+        
+    Returns:
+        Configured RedisValue class
+    """
+    from .client import RedisStorageClient
+    
+    storage_client = RedisStorageClient(redis_client, redis_key_prefix)
+    redis_dict = storage_client._redis_dict
```

Pylint (3.3.7)

[error] Instance of 'RedisStorageClient' has no '_redis_dict' member

(E1101)